debian/postinst

```diff
@@ -27,6 +27,38 @@ SYSTEMD_WANTS_AUTO_ATTACH_LINK="/etc/systemd/system/multi-user.target.wants/ua-a
 SYSTEMD_HELPER_ENABLED_AUTO_ATTACH_DSH="/var/lib/systemd/deb-systemd-helper-enabled/ua-auto-attach.service.dsh-also"
 SYSTEMD_HELPER_ENABLED_WANTS_LINK="/var/lib/systemd/deb-systemd-helper-enabled/multi-user.target.wants/ua-auto-attach.service"
 
+update_files_based_on_contract_delta() {
+  prev_release=$1
+  curr_release=$2
+
+    python3 -c "
+from uaclient.config import UAConfig
+from uaclient.contract import process_entitlements_delta
+
+past_entitlements=UAConfig(series='$prev_release').entitlements
+new_entitlements=UAConfig(series='$curr_release').entitlements
+
+process_entitlements_delta(
+   past_entitlements=past_entitlements,
+   new_entitlements=new_entitlements,
+   allow_enable=True
+)
+"
+}
+
+get_release_from_pkg_version(){
+    pkg_version=$1
+
+    if [[ $pkg_version =~ "14.04" ]]; then
```

Comment:
Could probably do this in a case statement (which makes me feel "safer" that string comparisons in POSIX shell as we already have examples of case/esac matching).